data/json/furniture.json

```diff
@@ -1973,6 +2037,7 @@
         "type" : "furniture",
         "id" : "f_anvil",
         "name": "anvil",
+        "description": "Craft some swords! Where's my lava?",
```

Comment:
Again, funny, and the `craft some swords` bit is better. Recommend you broaden the description to more clearly state what can be done with anvils.